### PR TITLE
[FIX] change logic of filter function

### PR DESCRIPTION
**Pull request by sarykemmoun**
I’ve changed the logic of the ‘isActivable’ function,
the function that filters the list in case that the switch component is off (not all tickets are chosen).
Instead of personal logic I’ve filtered it according to ticket properties
screenshot from 2017-10-23 13-40-10

### CODE REVIEW

**Comment by philjoseph** [comment]
The filter logic is that by default we display only tickets the user can act on them and not all tickets. The actionable tickets are all the open tickets where user can make followups and change a RDV if there is such one scheduled for the future. But it also includes "recently" closed tickets that the user can express satisfaction about. 
Before that change, we had an hard coded time for closed tickets but the time a ticket can be marked for satisfaction is configurable, so hard-coding was bas. Now with the change, we obtain from the server if a ticket be be marked for satisfaction so our filter is correct no matter what is configured.

**Comment by raphaelboukara** [comment]
@sarykemmoun it's not a fix, there was no bug, it's a feature. When @philjoseph defined a spec, you respected the spec. Now @philjoseph define a new spec => new feature.

File: js/components/Tickets/index.js [hunk 1]
```diff
@@ -17,13 +17,8 @@ class Tickets extends Component {
...
 
-    isActivable({ status, lastModifiedDate }) {
-        const modified = new Date(lastModifiedDate);
-        const now = new Date();
-        const diffMS = now - modified;
-        const diffD = diffMS / 1000 / 60 / 60 / 24;
-
-        return status !== 'Closed' || diffD < 15;
+    isActivable({ canAddFollowup, canReschedule, satisfactionStatus }) {
+        return canAddFollowup || canReschedule || satisfactionStatus === 1;
```
Comment (raphaelboukara) [inline js/components/Tickets/index.js]: OK for the fix, but there is a big refactor to do with this component.
This component don't need to care about filtering. We will speak about this component wednesday.

**Comment by sarykemmoun** [comment]
@raphaelboukara why does it not a bug?
the list not filtered correctly.

**Pull request merged**